swift.en.md

```diff
@@ -0,0 +1,340 @@
+# Swift coding style
+
+## Introduction
+
+This document defines the conventions for writing Swift code at Cookpad Inc.
```

Comment:
Oh, I copied it from Ruby version 😥 
https://github.com/cookpad/styleguide/blob/master/ruby.en.md#introduction

I'll get rid of Inc.